integrationtests/IntegrationTests.Shared/IntegrationTests.Shared.csproj

```diff
@@ -6,7 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="genesis.ensure" Version="2.0.0" />
-    <PackageReference Include="ReactiveUI" Version="*" />
+    <ProjectReference Include="..\..\src\ReactiveUI\ReactiveUI.csproj" />
```

Comment:
I also don't like this change. We might want to wait until the new `ReactiveUI.Avalonia` package ships, and then reference `ReactiveUI *` and `ReactiveUI.Avalonia *` instead of playing with paths.